components/service/sync-autofill/src/main/java/mozilla/components/service/sync/autofill/GeckoCreditCardsAddressesStorageDelegate.kt

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-package mozilla.components.browser.storage.sync
+package mozilla.components.service.sync.autofill
```

Comment:
Shouldn't the Gecko delegate live in the `browser-engine-gecko-*` components?

Reply:
This isn't actually the delegate wrapper around GV's Autcomplete (yet to be landed yet). We would add something equivalent to `GeckoLoginDelegateWrapper` in  `browser-engine-gecko-*`. This is equivalent to `GeckoLoginStorageDelegate`, but you could be right since this needs to be attached to the runtime. 

I don't know enough about the specifics of where this should live, but if we determine both `GeckoLoginStorageDelegate` and this file should be in `browser-engine-gecko-*`, let's file a new issue and move them both there.